### PR TITLE
fix: large hiccup during loading, now its spread along the loading bar

### PR DESCRIPTION
**Pull request by Kinerius**
## What does this PR change?

Fixes https://github.com/decentraland/unity-explorer/issues/563
Spreads along frames the re-enabling of terrain to fix a bug with the ColormapRenderer

## How to test the changes?

The loading screen should not get stuck at 92% for some seconds, instead, it should be spread out and the wait at 92% should be lower.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/Landscape/TerrainGenerator.cs [hunk 1]
```diff
@@ -194,6 +198,21 @@ public void SwitchVisibility(bool isVisible)
...
+        private async Task ReEnableTerrainAsync(AsyncLoadProcessReport processReport)
+        {
+            foreach (Terrain terrain in terrains)
+                terrain.enabled = false;
+
+            // we enable them one by one to avoid a super hiccup
+            for (var i = 0; i < terrains.Count; i++)
+            {
+                Terrain terrain = terrains[i];
+                terrain.enabled = true;
```
Comment (popuz) [inline Explorer/Assets/DCL/Landscape/TerrainGenerator.cs:210]: don't see the need for that variable. Is it possible to inline it?

File: Explorer/Assets/DCL/Landscape/TerrainGenerator.cs [hunk 2]
```diff
@@ -194,6 +198,21 @@ public void SwitchVisibility(bool isVisible)
             }
         }
 
+        private async Task ReEnableTerrainAsync(AsyncLoadProcessReport processReport)
+        {
```
Comment (popuz) [inline Explorer/Assets/DCL/Landscape/TerrainGenerator.cs:202]: I would still keep it in Utils and re-use for Worlds, so they don't deviate too much and we will be able to merge them in future. But not critical for now

**Review by popuz — APPROVED** [review]
👍

File: Explorer/Assets/DCL/Landscape/TerrainGenerator.cs (hunk 2 above)
Comment (Kinerius) [reply Explorer/Assets/DCL/Landscape/TerrainGenerator.cs]: ReEnableTerrainAsync

**Pull request merged**